lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -25,22 +26,23 @@ def self.parse(file)
     end
 
     def self.available?
-      `#{puppet_bin} help strings 2>&1`
-      if $?.success?
+      _stdout, _stderr, status = capture3([puppet_bin, 'help', 'strings'])
+      if status.success?
         return true
       else
-        raise KafoParsers::ParserNotAvailable.new("#{puppet_bin} does not have strings module installed")
+        raise KafoParsers::ParserNotAvailable.new("#{puppet_bin} does not have strings module installed. ")
```

Comment:
Any particular reason for the empty space at the end?
```suggestion
        raise KafoParsers::ParserNotAvailable.new("#{puppet_bin} does not have strings module installed.")
```